lib/cacheable/controller.rb

```diff
@@ -45,6 +45,8 @@ def response_cache(key_data=nil, version_data=nil, &block)
         h.cache_age_tolerance                            = cache_age_tolerance_in_seconds
         h.serve_unversioned                              = serve_unversioned_cacheable_entry?
         h.force_refill_cache                             = force_refill_cache?
+        h.cache_hit_server                               = ->(status, _, body) { render status: status, plain: body }
+        h.cache_hit_brower                               = ->(_, _, _) { head :not_modified }
```

Comment:
Rails becomes the default. Do we want this or assign lambdas conditionally? If a consumer of the gem does not use rails, it could be confusing to get an error message saying "head is not defined" instead of `configuration for cache_hit_server missing.`

Reply:
I don't think you'd use this module if you're not using rails.